webapp/lib/view_model.dart

```diff
@@ -134,9 +134,11 @@ class CodeSelector {
   static CodeSelector get activeCodeSelector => _activeCodeSelector;
   static set activeCodeSelector(CodeSelector activeCodeSelector) {
     _activeCodeSelector?.viewElement?.classes?.toggle('active', false);
+    _activeCodeSelector?.viewElement?.parent?.parent?.classes?.toggle('active', false);
```

Comment:
how about using getAncestors(_activeCodeSelector?.viewElement?).firstWhere((a) => a.classes.contains('message-row')?

I don't know if the above will work with null-aware operators, we might need to move this into an if.